kafka/source/pkg/apis/bindings/v1alpha1/kafka_conversion_test.go

```diff
@@ -147,6 +147,9 @@ func TestKafkaBindingConversionRoundTripV1alpha1(t *testing.T) {
 					Conditions: duckv1.Conditions{{
 						Type:   "Ready",
 						Status: "True",
+					}, {
+						Type:   "DoesNotReallyExistButHereItIsForTestingStatusCopy",
+						Status: "False",
```

Comment:
The old conversion test would've failed if we had a status with a condition other than `Ready` in it. I just added this imaginary condition to make sure we're not truncating the status conditions when doing the conversion.